dotnet/src/SemanticKernel/AI/OpenAI/Clients/OpenAIClientAbstract.cs

```diff
@@ -207,6 +207,18 @@ internal OpenAIClientAbstract(ILogger? log = null, IDelegatingHandlerFactory? ha
         }
     }
 
+    /// <summary>
+    /// Performs prompt normalization
+    /// </summary>
+    /// <param name="prompt">Prompt to normalize</param>
+    protected void NormalizePrompt(string prompt)
+    {
+        if (!string.IsNullOrWhiteSpace(prompt))
+        {
+            prompt = prompt.Replace("\r\n", "\n", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
This won't return the normalized value... it will just update in the context of this function, then be lost.

Reply:
Good catch, I also need to return it back.